packages/suite/src/components/suite/Settings/components/Analytics.tsx

```diff
@@ -11,16 +11,17 @@ import * as analyticsActions from '@suite-actions/analyticsActions';
 
 const mapStateToProps = (state: AppState) => ({
     locks: state.suite.locks,
-    analytics: state.suite.settings.analytics,
+    enabled: state.analytics.enabled,
 });
 
 const mapDispatchToProps = (dispatch: Dispatch) => ({
-    toggleAnalytics: bindActionCreators(analyticsActions.toggleAnalytics, dispatch),
+    init: bindActionCreators(analyticsActions.init, dispatch),
+    dispose: bindActionCreators(analyticsActions.dispose, dispatch),
 });
 
 export type Props = ReturnType<typeof mapStateToProps> & ReturnType<typeof mapDispatchToProps>;
 
-const Analytics = ({ locks, analytics, toggleAnalytics }: Props) => {
+const Analytics = ({ locks, enabled, init, dispose }: Props) => {
```

Comment:
inspiration for version 3 :)

you can have init, dispose and status served from `useAnalitics` hook, locks also are in hooks now, this way you will not need to use Container

Reply:
check it now.